src/main/scala-2.12/com/linagora/gatling/imap/scenario/ImapGetQuotaScenario.scala

```diff
@@ -12,5 +12,6 @@ object ImapGetQuotaScenario {
       .exec(imap("Connect").connect()).exitHereIfFailed
       .exec(imap("login").login("${username}", "${password}").check(ok))
       .exec(imap("select").select("INBOX").check(ok))
+      .exec(imap("setQuota").setQuota("INBOX (STORAGE 512)").check(no))
       .exec(imap("getQuota").getQuota("INBOX").check(bad))
```

Comment:
Could this no and bad be acceptable? (I do not fully understand set/get quota though).
James response:
```
a3 NO SETQUOTA You need the Full admin rights right to perform command SETQUOTA on mailbox Can not perform SETQUOTA commands.

a4 BAD GETQUOTA failed. No such mailbox.
```

Cyrus response:
```
a3 NO Permission denied

a4 NO Quota root does not exist
```

This does not come from the 2 commands themselves. Any suggestion about the scenario is welcome :')

Reply:
I don't know about Cyrus, but in James I would guess you can try to add a usersrepository.xml in james conf, with the `administratorId` declared to the user you would use in your integration test. Then that user should have admin rights to set up that quota? My guess, but not 100% sure :)

Reply:
Mhhh acceptable to me. James dissallow setting quota so...